src/haggadah/events.cljs

```diff
@@ -64,9 +74,9 @@
 (re-frame/reg-event-fx
  ::fetch-haggadot
  (fn [{:keys [db]} [_ {:keys [on-success on-error] :or {on-error :error}}]]
-   (println "Uid: " (:uid db))
    (if (:uid db)
-     {::fetch-collection! {:path ["users" (:uid db) "haggadot"]
+     {::query! {:path ["users" (:uid db) "haggadot"]
+                :order-by (fire/orderBy "createdAt" "desc")
```

Comment:
It seems that there are many effects or events that use the convention of receiving `on-success` and/or `on-error` and then be converted using `keyword->func`. Perhaps there's a way to extract this functionality to be reused.

Reply:
Could be done in  another PR....

Reply:
I'll make an issue for extracting a function which takes an `on-success` and an `on-error` function and which returns the functions after applying `keyword->func` on each one.